File: java/simplicity.java
import java.util.Arrays;
import java.util.Scanner;

public class simplicity {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		String input = scan.next();
		int[] char_counts = new int[26];
		for (char c : input.toCharArray())
			char_counts[c - 'a']++;
		Arrays.sort(char_counts);
		int count = 0;
		for (int i = 0; i < char_counts.length - 2; i++)
			count += char_counts[i];
		System.out.println(count);
		scan.close();
	}
}

File: ts/src/simplicity.ts
import readline = require('readline')

function getCharCounts (str: string): number[] {
  const charCounts = new Map<string, number>()
  for (const char of str) {
    charCounts.set(char, (charCounts.get(char) ?? 0) + 1)
  }
  return [...charCounts.values()]
}

function getMinimumCount (charCounts: number[]): number {
  charCounts.sort((a, b) => a - b)
  charCounts.pop()
  charCounts.pop()
  return charCounts.reduce((acc, curr) => acc + curr, 0)
}

readline
  .createInterface({
    input: process.stdin,
    output: process.stdout
  })
  .on('line', (line: string) => {
    const charCounts = getCharCounts(line)
    console.log(getMinimumCount(charCounts))
  })
  .on('close', () => {
    process.exit(0)
  })
